[Dario: make the patch to be applied with fuzz factor 0]
Signed-off-by: Dario Binacchi <dario.binacchi@amarulasolutions.com>
Common subdirectories: libfcgi-2.4.0.orig/cgi-fcgi and libfcgi-2.4.0/cgi-fcgi
Common subdirectories: libfcgi-2.4.0.orig/doc and libfcgi-2.4.0/doc
Common subdirectories: libfcgi-2.4.0.orig/examples and libfcgi-2.4.0/examples
Common subdirectories: libfcgi-2.4.0.orig/images and libfcgi-2.4.0/images
Common subdirectories: libfcgi-2.4.0.orig/include and libfcgi-2.4.0/include
Common subdirectories: libfcgi-2.4.0.orig/java and libfcgi-2.4.0/java
Common subdirectories: libfcgi-2.4.0.orig/libfcgi and libfcgi-2.4.0/libfcgi
diff -du libfcgi-2.4.0.orig/Makefile.am libfcgi-2.4.0/Makefile.am
--- libfcgi-2.4.0.orig/Makefile.am	2001-12-22 14:05:39.000000000 +0100
+++ libfcgi-2.4.0/Makefile.am	2014-07-19 16:56:21.935216540 +0200
@@ -10,7 +10,7 @@
 		aclocal.m4 m4/libtool.m4 m4/lt*.m4 Makefile.in install-sh missing ltmain.sh \
 		COPYING INSTALL
 
-SUBDIRS = libfcgi cgi-fcgi examples include
+SUBDIRS = libfcgi cgi-fcgi include
 
 include_HEADERS = fcgi_config.h
 
Common subdirectories: libfcgi-2.4.0.orig/perl and libfcgi-2.4.0/perl
Common subdirectories: libfcgi-2.4.0.orig/Win32 and libfcgi-2.4.0/Win32
